assets/js/base/hooks/shipping/use-shipping-rates.js

```diff
@@ -29,9 +30,15 @@ export const useShippingRates = () => {
 		( select ) => select( storeKey ).areShippingRatesLoading(),
 		[]
 	);
+	const decodedShippingAddress = {};
+	Object.keys( shippingAddress ).forEach( ( key ) => {
+		decodedShippingAddress[ key ] = decodeEntities(
+			shippingAddress[ key ]
+		);
+	} );
```

Comment:
This looks like something that should be implemented in `useShippingAddress` instead of `useShippingRates`?

Reply:
Absolutely! Done in d237e4d6c6c2d80fb41cf6eb2dedf54467f865dc.